jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AbstractActions.kt

```diff
@@ -16,7 +16,11 @@ import javax.swing.Icon
  *
```

Comment:
This whole file just looks like formatting.

Reply:
Ya, will revert